docs/source/release_notes.rst

```diff
@@ -25,6 +25,11 @@ Release Notes
         * Pin graphviz version for windows builds :pr:`1847`
         * Unpin graphviz version for windows builds :pr:`1851`
 
+.. warning::
+
+    **Breaking Changes**
```

Comment:
This is only listed as a breaking change but not in the other sections of the release notes? :O Is this intentional 👀

Reply:
I originally had it in both but then felt it was redundant. Do we have a guideline in place for this?

Reply:
To date, we've listed in both places. And a single PR could contain multiple lines of change in release notes. However for something this small, I think this is fine.